payments/index.md

```diff
@@ -157,10 +157,9 @@ After creating a payment, you can:
  All actions after creating the payment can be done by using our APIs, or from
  our admin tool. `abort` is only available when using APIs.
 
-Please visit our [demoshop][demoshop] to see our Payment Menu and Redirect
+Please visit our [demoshop](https://ecom.externalintegration.payex.com/pspdemoshop) to see our Payment Menu and Redirect
```

Comment:
As this link is not within an include, do we need to inline it?

Reply:
I was a bit too aggressive in my changes.